_episodes/03.md

```diff
@@ -15,39 +15,32 @@ keypoints:
 - With GitHub Desktop, we can do everything we can on the web 
 - A **remote** links a repository on your computer to a remote repository on a server
 - You can **publish** and **push** changes made on your computer to a remote server
-- You can **pull** changes made on a remote server to your computer
-- Additionally, all of this can be done using a command line interface 
+- You can **pull** changes made on a remote server to your computer 
```

Comment:
I don't think this is part of the current version on the episode?

Reply:
Indeed, I've removed that key point